File: src/main/java/de/bypixeltv/redivelocity/utils/DateUtils.java
package de.bypixeltv.redivelocity.utils;

import jakarta.inject.Singleton;
import java.text.SimpleDateFormat;
import java.util.Date;

@Singleton
public class DateUtils {

    public static String asDateString(long timestamp) {
        SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yyyy HH:mm:ss");
        return sdf.format(new Date(timestamp));
    }
}

File: src/main/kotlin/de/bypixeltv/redivelocity/utils/DateUtils.kt
package de.bypixeltv.redivelocity.utils

import jakarta.inject.Singleton
import java.text.SimpleDateFormat
import java.util.*

@Singleton
class DateUtils

fun Long.asDateString(): String {
    val sdf = SimpleDateFormat("dd/MM/yyyy HH:mm:ss")
    return sdf.format(Date(this))
}